pkg/package.mk

```diff
@@ -0,0 +1,17 @@
+.PHONY: tag push
+default: push
+
+ORG?=linuxkit
+HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
+NETWORK=0
+
+tag: $(DEPS)
+ifeq ($(NETWORK), 1)
+	docker build --squash --no-cache -t $(ORG)/$(IMAGE):$(HASH) .
```

Comment:
Also we should not really need `--no-cache` as everything in `pkg` should be reproducible.